client/src/test/scala/uk/gov/ons/addressIndex/client/AddressIndexClientTest.scala

```diff
@@ -1,10 +1,9 @@
 package uk.gov.ons.addressIndex.client
 
-import javax.inject.{Inject, Singleton}
 import org.scalatest.FlatSpec
-import play.api.libs.ws.WSClient
 
 class AddressIndexClientTest extends FlatSpec {
-  ignore "no tests" in {
+  ignore should "no tests" in {
```

Comment:
No tests, wanted something to come up when you run sbt test though, telling you theres no tests, hence ignore